app/models/concerns/conclusion_final_reviews/defaults.rb

```diff
@@ -7,6 +7,23 @@ module ConclusionFinalReviews::Defaults
     after_initialize :set_defaults, if: :new_record?
   end
 
+  def duplicate_annexes_and_images_from_draft
+    draft = ConclusionDraftReview.where(review_id: review_id).first
+
+    if draft
+      draft.annexes.each do |annex|
+        new_annex = annexes.build annex.attributes.dup.merge('id' => nil)
+
+        annex.image_models.each do |image_model|
+          new_image = ImageModel.new
+          new_image.image = File.open image_model.image.file.file
```

Comment:
```suggestion
          new_image.image = File.open image_model.image.path
```
Debería funcionar y no se ve tan raro... jeje.

Reply:
perfecto, va este